redundancy/redundancy.go

```diff
@@ -0,0 +1,100 @@
+package redundancy
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"sync"
+)
+
+var log = logger.GetOrCreate("redundancy")
+
+// maxRoundsOfInactivityAccepted defines the maximum rounds of inactivity accepted, after which the master or lower
+// level redundancy machines will be considered inactive
+const maxRoundsOfInactivityAccepted = 5
```

Comment:
why is this so lower? In the discussed specs we wanted to disable the automatic block signing by defining a very high value for this.

Reply:
This constant is multiplied with the level given by the flag, so if we want to turn off this mechanism we should run the node with the level set to some of 100000 for example. This constant actually will define the difference from two backup machines by multiply the given level by 5

Reply:
Right, didn't saw that. So, in order to disable automatic signing, we just have to put this to a large value. :+1: